src/autopas/LogicHandler.h

```diff
@@ -504,22 +512,34 @@ class LogicHandler {
    */
   [[nodiscard]] unsigned long getNumberOfParticlesHalo() const { return _numParticlesHalo; }
 
+  /**
+   * Check if other autotuners for any other interaction types are still in a tuning phase.
+   * @param interactionType
+   * @return bool whether other tuners are still tuning.
+   */
+  bool checkTuningStates(InteractionTypeOption::Value interactionType) {
+    return std::any_of(std::begin(_autoTunerRefs), std::end(_autoTunerRefs), [&](const auto &entry) {
+      return entry.first == interactionType ? false : entry.second->inTuningPhase();
+    });
```

Comment:
```suggestion
    // Goes over all pairs in _autoTunerRefs and returns true as soon as one is `inTuningPhase()`.
    // The tuner associated with the given interaction type is ignored.
    return std::any_of(std::begin(_autoTunerRefs), std::end(_autoTunerRefs), [&](const auto &entry) {
      return entry.first == interactionType ? false : entry.second->inTuningPhase();
    });
```
Correct me if I misunderstood.